src/Sylius/Bundle/CoreBundle/Command/InstallDatabaseCommand.php

```diff
@@ -54,6 +58,10 @@ protected function execute(InputInterface $input, OutputInterface $output): void
         $this->runCommands($commands, $output);
         $outputStyle->newLine();
 
-        $this->commandExecutor->runCommand('sylius:install:sample-data', [], $output);
+        $parameters = [];
+        if ($suite) {
```

Comment:
Can you make explict comparition here? What do we accept as a proper value? Not empty string?